components/compositing/compositor.rs

```diff
@@ -739,15 +721,17 @@ impl IOCompositor {
                     return;
                 };
 
-                let offset = LayoutVector2D::new(point.x, point.y);
                 let Some(offset) = pipeline_details
                     .scroll_tree
-                    .set_scroll_offsets_for_node_with_external_scroll_id(
+                    .set_scroll_offset_for_node_with_external_scroll_id(
                         external_scroll_id,
-                        -offset,
+                        offset,
```

Comment:
Will it still compute correctly here? I saw `set_scroll_offset_for_node_with_external_scroll_id` doesn't change anything